package.json

```diff
@@ -41,6 +42,7 @@
     "web3": "1.3.4"
   },
   "peerDependencies": {
-    "puppeteer": ">5"
+    "puppeteer": ">7",
+    "node-stream-zip": "^1.13.0"
```

Comment:
Ain0't this suppose to be prod dependency?

Reply:
:man_shrugging: we want to lock the user to a specific dependency?